CONTRIBUTING.md

```diff
@@ -16,13 +16,36 @@ Lastly, run the test command from the console:
 
     npm test
 
+
 ## Deploying
 
-- `npm run prepublish && npm run docs`
-- Update the version key in [package.json](https://github.com/mapbox/mapbox-gl-geocoder/blob/master/package.json#L3)
-- Outline changes in [CHANGELOG.md](https://github.com/mapbox/mapbox-gl-geocoder/blob/master/CHANGELOG.md)
-- Commit and push
-- `git tag -a vX.X.X -m 'vX.X.X'`
-- `git push --tags`
-- `npm publish`
-- Update version number in GL JS examples ([one](https://github.com/mapbox/mapbox-gl-js/blob/publisher-production/docs/pages/example/mapbox-gl-geocoder.html), [two](https://github.com/mapbox/mapbox-gl-js/blob/publisher-production/docs/pages/example/point-from-geocoder-result.html), [three](https://github.com/mapbox/mapbox-gl-js/blob/publisher-production/docs/pages/example/mapbox-gl-geocoder-outside-the-map.html), [four](https://github.com/mapbox/mapbox-gl-js/blob/publisher-production/docs/pages/example/mapbox-gl-geocoder-limit-region.html), [five](https://github.com/mapbox/mapbox-gl-js/blob/publisher-production/docs/pages/example/mapbox-gl-geocoder-accept-coordinates.html), [six](https://github.com/mapbox/mapbox-gl-js/blob/publisher-production/docs/pages/example/mapbox-gl-geocoder-proximity-bias.html), [seven](https://github.com/mapbox/mapbox-gl-js/blob/publisher-production/docs/pages/example/forward-geocode-custom-data.html))
+Follow this deploy process after all changes for the release are merged into master. You will copy and paste this checklist in the comment of the release pull request.
+
+```
+## Release checklist
+
+1. Create a branch off `master` and a pull request with the following changes. Copy this checklist in the comment of the pull request.
+    - [ ] Update the [CHANGELOG.md](https://github.com/mapbox/mapbox-gl-geocoder/blob/master/CHANGELOG.md) by comparing the last release and what is on `master`. In the changelog, replace the `master` heading with the to-be-released stable version.
+    - [ ] Update the version number in `package.json` and `package.lock.json`.
+3. Request a review PR and then merge it into `master`.
+4. Tag the release and start the build.
+    - [ ] Make sure you've pulled in all changes to `master` locally.
+    - [ ] Build the release with `npm run prepublish && npm run docs`
```

Comment:
Do you think doc changes should be committed as part of each PR in the lead up to a release, or left out to the release PR?

I think it's better to commit them as we go for each feature, that way you have complete docs for any given commit.

Reply:
> I think it's better to commit them as we go for each feature, that way you have complete docs for any given commit.

I agree, I think we should encourage PRs to update the changelog before merging to help keep track of what's in master. I included this as a step in the pull request template.